parser/parser.y

```diff
@@ -6327,8 +6327,9 @@ UpdateStmt:
 		}
 		st := &ast.UpdateStmt{
 			LowPriority:	$2.(bool),
-			TableRefs:	&ast.TableRefsClause{TableRefs: refs},
-			List:		$6.([]*ast.Assignment),
+			TableRefs:	    &ast.TableRefsClause{TableRefs: refs},
+			List:		    $6.([]*ast.Assignment),
+			Ignore:         $3.(bool),	
```

Comment:
please fix the indent

Reply:
What's the specification of the parser.y file. My vim automatically format the indent when saving the write.